canvasapi/assignment.py

```diff
@@ -160,6 +160,32 @@ def get_peer_reviews(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def get_provisional_grades_status(self, user, **kwargs):
+        """
+        Tell whether the student's submission needs one or more provisional grades.
+
+        :calls: `GET /api/v1/courses/:course_id/assignments/:assignment_id/provisional_grades/
+            status \
+        <https://canvas.instructure.com/doc/api/all_resources.html#method.provisional_grades.status>`_
+
+        :param user: The object or ID of the related user
+        :type user: :class:`canvasapi.user.User` or int
+
+        :rtype: bool
+        """
+        user_id = obj_or_id(user, "user", (User,))
+        request = self._requester.request(
+            "GET",
+            "courses/{}/assignments/{}/provisional_grades/status/{}".format(
```

Comment:
The URL for this endpoint doesn't use the ID. Instead, `student_id` should be passed along as a parameter via `_kwargs`. It must be called `student_id` and not `user_id`.

It should be:
```courses/{}/assignments/{}/provisional_grades/status```